src/main/antlr4/de/zalando/beard/BeardParser.g4

```diff
@@ -178,4 +179,9 @@ locals [NewLine result]
 text
 locals [Text result]
     : TEXT
```

Comment:
May be then just make them allowed by removing from the line 9 of BeardLexer.g4 converting it into